packages/types/src/clerk.ts

```diff
@@ -277,38 +277,90 @@ export interface Clerk {
   navigate: CustomNavigation;
 
   /**
-   * Redirects to the configured sign in URL. Retrieved from {@link environment}.
+   *
+   * Decorates the provided url with the auth token for development instances.
+   *
+   * @param {string} to
+   */
+  buildUrlWithAuth(to: string): string;
+
+  /**
+   * Returns the configured url where <SignIn/> is mounted.
+   *
+   * @param opts A {@link RedirectOptions} object
+   *
+   */
+  buildSignInUrl(opts?: RedirectOptions): string;
+
+  /**
+   * Returns the configured url where <SignUp/> is mounted.
+   *
+   * @param opts A {@link RedirectOptions} object
+   *
+   */
+  buildSignUpUrl(opts?: RedirectOptions): string;
+
+  /**
+   * Returns the url where <UserProfile /> is mounted.
+   */
+  buildUserProfileUrl(): string;
+
+  /**
+   * Returns the configured url where <CreateOrganization /> is mounted.
+   */
+  buildCreateOrganizationUrl(): string;
+
+  /**
+   * Returns the configured url where <OrganizationProfile /> is mounted.
+   */
+  buildOrganizationProfileUrl(): string;
+
+  /**
+   * Returns the configured home URL of the instance.
+   */
+  buildHomeUrl(): string;
+
+  /**
+   *
+   * Redirects to the provided url after decorating it with the auth token for development instances.
+   *
+   * @param {string} to
+   */
+  redirectWithAuth(to: string): Promise<unknown>;
+
+  /**
+   * Redirects to the configured URL where <SignIn/> is mounted.
    *
    * @param opts A {@link RedirectOptions} object
    */
   redirectToSignIn(opts?: RedirectOptions): Promise<unknown>;
 
   /**
-   * Redirects to the configured sign up URL. Retrieved from {@link environment}.
+   * Redirects to the configured URL where <SignUp/> is mounted.
```

Comment:
:question: Are these comments correct? What if I've configured my display_config.sign_up_path to a custom path and I don't mount the `<SignUp/>` component?

I'll be redirected to the configured sign up URL.

Same for the rest of the redirect functions.

Reply:
Valid point.

Reply:
The comment has been updated.